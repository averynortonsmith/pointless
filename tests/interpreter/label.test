
output = [a, b, c, d] |> println

a = Foo(1, 2, 3)
b = Foo
c = Foo {}
d = Foo {x = 123}