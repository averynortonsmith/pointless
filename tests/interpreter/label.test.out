[Foo(1, 2, 3), Foo, Foo {}, Foo {x = 123}]
